NGCHM/WebContent/javascript/custom/custom.js

```diff
@@ -53,6 +53,12 @@ linkouts.addPanePlugin ({
     linkouts.addSubtype ("bio.gene.entrez", "bio.gene.entrezid");
 
     linkouts.describeTypes([
+	{ typeName: "bio.genome.position",
+	  displayName: "Genome Position/Range",
+	  description: "Genome position/range",
```

Comment:
I find the slash confusing.  Isn't it "Genome Position Range" or simply "Genome Range"?  All the same, I searched the code and see this displayName and description are never accessed.